tests/queries/0_stateless/00926_adaptive_index_granularity_replacing_merge_tree.reference

```diff
@@ -1,7 +1,7 @@
 4
 4
 8
-7
+8
```

Comment:
Change of test is ok, because it checks number of written marks. Actually it should be 8. Horizontal merge also creates 8 marks in that case.